gulpfile.js

```diff
@@ -1,67 +1,66 @@
-var debug = true;
-
+var autoprefixer = require('gulp-autoprefixer');
+var babelify = require('babelify');
+var browserify = require('browserify');
+var buffer = require('vinyl-buffer');
+var cache = require('gulp-cache');
+var del = require('del');
+var eslint = require('gulp-eslint');
 var gulp = require('gulp');
 var gutil = require('gulp-util');
-var sass = require('gulp-sass');
 var imagemin = require('gulp-imagemin');
-var sass = require('gulp-sass');
-var del = require('del');
-var runSequence = require('run-sequence');
+var minifycss = require('gulp-minify-css');
+var notify = require('gulp-notify');
 var rename = require('gulp-rename');
-var browserify = require('browserify');
-var babelify = require('babelify');
+var runSequence = require('run-sequence');
+var sass = require('gulp-sass');
```

Comment:
Ignore all the diff churn. I just A-Zed the `require()` statements and removed the dupe `sass` var require.